src/bot.py

```diff
@@ -2,6 +2,7 @@
 from discord.ui import Modal, Select, InputText, Button
 import re
 import logging
+from dotenv import load_dotenv
```

Comment:
We don't want to use dotenv because this is expected to be running in a context where the environment variables are setup. That can be done via docker or locally by sourcing the .env